test/integration/controller_test.go

```diff
@@ -76,397 +76,207 @@ const (
 // - unbind
 // - deprovision
 // - delete broker
-//
-// ...using purely synchronous provision/deprovision.
-func TestBasicFlowsSync(t *testing.T) {
-	_, catalogClient, _, _, _, _, shutdownServer, shutdownController := newTestController(t, fakeosb.FakeClientConfiguration{
-		CatalogReaction: &fakeosb.CatalogReaction{
```

Comment:
I do kind of like having these all specified out as a struct vs the set a field of a field of a field of a struct I do not see. It doesn't look like the newController needs the osbclient that's passed in to actually set anything up, so I think the shift in paradigm is good.